xbmc/platform/linux/powermanagement/X11DPMSSupport.cpp

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
this object is owned by windowing and the file is platform specific. I would move it to windowing.